pkg/deploy/defaults.go

```diff
@@ -57,3 +59,15 @@ const (
 	DefaultSecurityContextFsGroup       = "1724"
 	DefaultSecurityContextRunAsUser     = "1724"
 )
+
+func DefaultPullPolicyFromDockerImage(dockerImage string) string {
```

Comment:
I am surprised about that @nickboldt didn't comment about your usage of the `docker` word here :-)

Reply:
haha. He still can do a PR afterwards if he prefers other words :-)